tests/unit/helpers_test.sh

```diff
@@ -67,7 +67,7 @@ function test_no_function_is_executed_with_execute_function_if_exists() {
 }
 
 function test_unsuccessful_unsetIfExists() {
-  assert_general_error "$(Helper::unsetIfExists "fake_function")"
+  assert_successful_code "$(Helper::unsetIfExists "fake_function")"
```

Comment:
Isn't the test checking the unsuccessful behavior?

Reply:
In my opinion the `unsetIfExists` function should not return any error in case the function doesn't exists. As the function name said, it's only unset if it exists.

Reply:
Oh, good point! I agree